po/fr.po

```diff
@@ -0,0 +1,196 @@
+# French translations for com.github.torikulhabib.nino package.
+# Copyright (C) 2019 THE com.github.torikulhabib.nino'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.torikulhabib.nino package.
+# NathanBnm, 2019.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.torikulhabib.nino\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-11-17 10:36+0700\n"
+"PO-Revision-Date: 2019-01-19 21:07+0100\n"
+"Last-Translator: NathanBnm\n"
+"Language-Team: Français\n"
+"Language: fr\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:7
+#: data/com.github.torikulhabib.nino.desktop.in.in:3
+msgid "Nino"
+msgstr "Nino"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:8
+#: data/com.github.torikulhabib.nino.desktop.in.in:4
+msgid "A network monitor widget"
+msgstr "Un widget de moniteur réseau"
+
+#: data/com.github.torikulhabib.nino.desktop.in.in:5
+msgid "Network Monitor"
+msgstr "Moniteur réseau"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:10
+msgid "An internet speed monitor apps."
+msgstr "Une application de surveillance de la vitesse d'Internet."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:11
+msgid "Features:"
+msgstr "Fonctionnalités :"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:13
+msgid "Small window application features"
+msgstr "Toutes les fonctionnalités de l'application dans une petite fenêtre"
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:14
+msgid "Alert when no internet connection is connected."
+msgstr "Soyez alerté lorsqu'aucune connexion Internet n'est établie."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:15
+msgid "Many color theme choices available."
+msgstr "Plusieurs choix de couleurs de thème disponibles."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:16
+msgid "Lock position on the desktop."
+msgstr "Verrouillez la position sur le bureau."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:17
+msgid "Save the position of the widget."
+msgstr "Enregistrez la position du widget"
```

Comment:
I suppose this translation should end with period.

Reply:
You're right I will fix this